Firestore/core/src/remote/grpc_streaming_reader.h

```diff
@@ -60,7 +62,9 @@ class GrpcStreamingReader : public GrpcCall, public GrpcStreamObserver {
    * results of the call. If the call fails, the `callback` will be invoked with
    * a non-ok status.
    */
-  void Start(Callback&& callback);
+  void Start(size_t key_count,
```

Comment:
Can you update the comment? We should also not use the term `key` but rather something like`expected_response_count`.